### ISSUE TITLE
Ask textual feedback only if bad review 

### ISSUE THREAD

**Issue by BastinFlorian** [trubrics/streamlit-feedback#13]
# Description 

Is it possible to ask for textual feedback only if the review is negative or is not the best emoji review ?

**Comment by jeffkayne** [comment]
Yes could be a nice feature. Is this something that you would be able to contribute?

**Comment by SoumyaMukhija** [comment]
I can take this one!
